websites/L/Light Novel Pub/presence.ts

```diff
@@ -0,0 +1,77 @@
+const presence = new Presence({ clientId: "952854435711819796" }),
+	browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const { title } = document,
+		{ href, pathname } = document.location,
+		presenceData: PresenceData = {
+			largeImageKey: "logo",
+			details: "Browsing...",
+			startTimestamp: browsingStamp
+		};
+
+	if (pathname.startsWith("/novel/")) {
+		presenceData.largeImageKey = document
+			.querySelector('meta[property="og:image"]')
+			.getAttribute("content");
+		presenceData.smallImageKey = "logo";
+		presenceData.startTimestamp = browsingStamp;
+		presenceData.buttons = [
+			{
+				label: "",
```

Comment:
?

Reply:
i edited it again down below cuz i usually set the button's label just the same with either `.Details` or `.State` so... should i not?